R/scale-view.r

```diff
@@ -87,6 +109,8 @@ ViewScale <- ggproto("ViewScale", NULL,
   # map, rescale, and make_title need a reference
```

Comment:
I'm somewhat conflicted on the naming and implementation of `ViewScale`. At a glance it looks like a `Scale` object with many (all?) of the same methods etc. The file naming also suggest it is a specific scale type. But obviously this is not the case. At the very least we should rename the file to put it away from the other scale implementation, but can you also share a bit of the reasoning for this design (you may have already discussed this with Hadley, but I'm curious as well)

Reply:
ok